src/extract_bootstrap_from_bundles.js

```diff
@@ -11,7 +11,7 @@ const extractBootstrapFromBundles = async ({
   const gettingBundles = makeProgress('Getting bundles');
   let scriptsContents;
   try {
-    scriptsContents = await Promise.all(bundleUrls.map((url) => httpsGet(url, projectName)));
+    scriptsContents = (await Promise.allSettled(bundleUrls.map((url) => httpsGet(url, projectName)))).map(({value}) => value || "");
```

Comment:
Will httpsGet return { value } when rejecting? Why Promise.allSettled?

Reply:
Yes, this works even if the page returns a 404. Return value will be something like `[{status: "fulfilled", value: 99}, {status: "rejected",  reason: Error: an error}]`